### PR TITLE
Removed LogLuvLoader

### PR DESCRIPTION
**Pull request by mrdoob**
Related: #29143

@Mugen87 Now that we have UltraHDR, I think we can remove this loader?

### CODE REVIEW

**Comment by Mugen87** [comment]
Are you planning to remove `RGBMLoader` as well?

If so, is it possible to convert RGBM or LogLuv HDR textures to UltraHDR somehow? The thing is if we remove those loaders, it's not possible to import such HDR textures into `three.js` anymore (unless apps maintain the loaders for themselves).

File: examples/files.json [hunk 1]
```diff
@@ -121,7 +121,6 @@
 		"webgl_loader_texture_hdr",
 		"webgl_loader_texture_ktx",
 		"webgl_loader_texture_ktx2",
-		"webgl_loader_texture_logluv",
```
Comment (Mugen87) [inline examples/files.json:124]: The respective E2E screenshot also needs to be removed, otherwise the CI complains.

**Comment by WestLangley** [comment]
>Are you planning to remove RGBMLoader as well?

Also, is the plan to rename `RGBELoader` to `HDRLoader` -- for loading Radiance .hdr files -- or to remove `RGBELoader`, too?

**Comment by Mugen87** [comment]
IMO, we should not remove `RGBELoader`. 

If we follow that logic, we could apply it in context of 3D assets as well and basically remove almost every loader except for `GLTFLoader`.

**Comment by sommaichaisit** [comment]
> Related: #29143
> 
> @Mugen87 Now that we have UltraHDR, I think we can remove this loader?

**Comment by mrdoob** [comment]
> > Are you planning to remove RGBMLoader as well?
> 
> Also, is the plan to rename `RGBELoader` to `HDRLoader` -- for loading Radiance .hdr files -- or to remove `RGBELoader`, too?

Renaming `RGBELoader` to `HDRLoader` sounds good to me.

File: examples/files.json (hunk 1 above)
Comment (mrdoob) [reply examples/files.json:124]: Done!

**Comment by mrdoob** [comment]
@Mugen87 

> Are you planning to remove `RGBMLoader` as well?

Probably too?

**Comment by Mugen87** [comment]
Okay, I'm fine with the removal of both loaders. 

I suspect they are only sporadically used because of their exotic character so there should not be that much of complains. There are better alternatives as well.

**Pull request merged**